kong/db/migrations/core/019_320_to_330.lua

```diff
@@ -0,0 +1,35 @@
+return {
+  postgres = {
+    up = [[
+      DO $$
+          BEGIN
+          ALTER TABLE IF EXISTS ONLY "plugins" ADD "updated_at" TIMESTAMP WITH TIME ZONE DEFAULT timezone('UTC'::text, ('now'::text)::timestamp(0) with time zone);
```

Comment:
```suggestion
          ALTER TABLE IF EXISTS ONLY "plugins" ADD "updated_at" TIMESTAMP WITH TIME ZONE DEFAULT (CURRENT_TIMESTAMP(0) AT TIME ZONE 'UTC');
```

Does that work too? It would be more inline on what we have elsewhere. And same for the rest.